### PR TITLE
Add simple chatbot test

### PR DESCRIPTION
**Pull request by jerpint**

### CODE REVIEW

File: tests/test_chatbot.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        text_before_prompt=(
+            """You are a slack chatbot assistant answering technical questions about huggingface transformers, a library to train transformers in python.\n"""
+            """Make sure to format your answers in Markdown format, including code block and snippets.\n"""
+            """Do not include any links to urls or hyperlinks in your answers.\n\n"""
+            """Now answer the following question:\n"""
+        ),
+    )
+    chatbot = Chatbot(hf_transformers_cfg)
+    answer = chatbot.process_input("What is a transformer?")
+    assert type(answer) is str
```
Comment (hbertrand) [inline tests/test_chatbot.py:33]: should be `isinstance`

File: tests/test_chatbot.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import os
+from pathlib import Path
+
+from buster.chatbot import Chatbot, ChatbotConfig
+
+TEST_DATA_DIR = Path(__file__).resolve().parent / "data"
+DOCUMENTS_FILE = os.path.join(str(TEST_DATA_DIR), "document_embeddings_huggingface_subset.tar.gz")
```
Comment (hbertrand) [inline tests/test_chatbot.py:7]: This will break every time we change the way we store data, which is often since we're in the middle of a refactor. The proper way would be to create a mock file in a fixture.

**Review by hbertrand — APPROVED** [review]
We can merge if you just want a quick test, but this will need to be done properly soon.

File: tests/test_chatbot.py (hunk 2 above)
Comment (jerpint) [reply tests/test_chatbot.py:7]: ok, good idea, i can create a mock file but it will have to create the embeddings every time, not a big deal but i'll need a quick script from you

File: tests/test_chatbot.py (hunk 1 above)
Comment (jerpint) [reply tests/test_chatbot.py]: fixed

File: tests/test_chatbot.py (hunk 2 above)
Comment (hbertrand) [reply tests/test_chatbot.py:7]: Do we need real embeddings though? We can (should?) monkeypatch `get_embedding` and `self.documents.retrieve`, since we're testing the chatbot.py code, not the rest.

Comment (hbertrand) [reply tests/test_chatbot.py:7]: Do we need real embeddings though? We can (should?) monkeypatch `get_embedding` and `self.documents.retrieve`, since we're testing the chatbot.py code, not the rest.

Comment (jerpint) [reply tests/test_chatbot.py:7]: the original intent of this one specific test in this PR was to have it be as close as possible to the actual chatbots such taht when we propose new changes, we are fairly confident things didn't break in our deployed apps. I do agree that we need a whole suite of tests, but this one is just the "bare minimum" to ensure things work

Comment (hbertrand) [reply tests/test_chatbot.py:7]: then let's just merge it and fix later
